grammarinator/generate.py

```diff
@@ -242,6 +242,9 @@ def restricted_float(value):
     parser.add_argument('-c', '--cooldown', default=1.0, type=restricted_float, metavar='NUM',
                         help='cool-down factor defines how much the probability of an alternative should decrease '
                              'after it has been chosen (interval: (0, 1]; default: %(default)f).')
+    parser.add_argument('-s', '--seed', type=int, metavar='NUM',
+                        help='defines the seed of random, also use "-j 1" combined with this option to make generating'
+                             'repeatable, is not set by default (every generating should be different)')
```

Comment:
'initialize random number generator with fixed seed (not set by default; noneffective if parallelization is enabled)' might be a better help string.